components/admin/users/AdminUsersTable.tsx

```diff
@@ -56,9 +57,9 @@ const AdminOption: React.FC<AdminOptionProps> = ({
     }
   }
 
-  const changeButton = async () => {
-    await changeRights(mutationVariable)
-    const newUsers = [...users]
+  const changeButton = () => {
+    changeRights(mutationVariable)
+    const newUsers = _.cloneDeep(users)
```

Comment:
Fixed a bug, right now if you open this page and try to add admin rights it will throw an error, something about `newUsers[usersIndex].isAdmin` is readonly. Deepcloning fixed it.

Reply:
I tried to add and remove admin rights on the website and its working perfectly. Can you take a screenshot of the error?

Reply:
![adminError](https://user-images.githubusercontent.com/3845012/107861519-53ad2b80-6e57-11eb-829b-67c93c62c5db.png)
I haven't thought about why it happens, I guess spread didn't go deep enough and we're trying to reassign original users value which is used in a hook?

Reply:
This is really strange, because I just tested on my local, and it doesn't work -- I get the same error you do. But in production it works fine. And before, when I created this page, it worked fine on local. Maybe an update to our dependencies caused this error? Weird... thanks for noticing the error and fixing it :)

Reply:
More Info:
I tried it with our production build (`yarn start -p 3000`), and it worked fine.